api-admin/src/main/java/bio/terra/pearl/api/admin/service/notifications/NotificationConfigExtService.java

```diff
@@ -133,6 +139,46 @@ public NotificationConfig create(
     return create(newConfig, studyEnvironment, portalEnvironment);
   }
 
+  /**
+   * Deletes the config specified by id, ensuring it belongs the appropriate study and environment
+   */
+  @Transactional
+  public void delete(
+      AdminUser user,
+      String portalShortcode,
+      String studyShortcode,
+      EnvironmentName environmentName,
+      UUID configId) {
+    authUtilService.authUserToPortal(user, portalShortcode);
+    authUtilService.authUserToStudy(user, portalShortcode, studyShortcode);
+    PortalEnvironment portalEnvironment =
+        portalEnvironmentService
+            .findOne(portalShortcode, environmentName)
+            .orElseThrow(PortalEnvironmentMissing::new);
+    StudyEnvironment studyEnvironment =
+        studyEnvironmentService
+            .findByStudy(studyShortcode, environmentName)
+            .orElseThrow(StudyEnvironmentMissing::new);
+
+    // make sure it exists/has the appropriate props
+    Optional<NotificationConfig> configOpt = notificationConfigService.find(configId);
+    if (configOpt.isEmpty()) {
+      throw new NotFoundException("Could not find notification config.");
+    }
+
+    configOpt.ifPresent(
+        config -> {
+          // check if it's in the right portal/study env
+          verifyNotificationConfig(config, portalEnvironment, studyEnvironment);
+          notificationConfigService.attachTemplates(List.of(config));
+
```

Comment:
Ah, this is my fault for not thinking through the ticket carefully.  While from a user perspective we want to delete notification configs, we don't want to *delete* notification configs from the database, for exactly the reason that they might have records of email correspondence referencing them.  Instead, we want to deactivate them.  Fortunately, that's much simpler

```
    config.setActive(false);
    notificationConfigService.update(existing);
```

Sorry you had to spend so much time working through the notification cascades -- but it's definitely not lost time since we both learned/relearned fun stuff about spring injection of test services :)